api/openapi-spec/swagger.json

```diff
@@ -1264,347 +1321,11 @@
        "description": "TimeoutSeconds for the list/watch call.",
        "name": "timeoutSeconds",
        "in": "query"
-      }
-     ],
-     "responses": {
-      "200": {
-       "description": "OK",
-       "schema": {
-        "$ref": "#/definitions/v1.VirtualMachineList"
-       }
-      },
-      "404": {
-       "description": "Not Found"
-      },
-      "default": {
-       "description": "OK",
-       "schema": {
-        "$ref": "#/definitions/v1.VirtualMachineList"
-       }
-      }
-     }
-    }
-   },
-   "/apis/kubevirt.io/v1alpha1/watch/migrations": {
```

Comment:
When I look here: https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json, I can still see the /watch/ endpoints.

Reply:
Oh, I am sorry ... I don't  know on what I was looking at then.